addons/web_editor/static/src/js/editor/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -54,8 +54,18 @@ describe('Copy', () => {
                     const clipboardData = new DataTransfer();
                     triggerEvent(editor.editable, 'copy', { clipboardData });
                     window.chai.expect(clipboardData.getData('text/plain')).to.be.equal('abc\nefg');
-                    window.chai.expect(clipboardData.getData('text/html')).to.be.equal('abc<br>efg');
-                    window.chai.expect(clipboardData.getData('text/odoo-editor')).to.be.equal('abc<br>efg');
+                    window.chai.expect(clipboardData.getData('text/html')).to.be.equal('<p>abc<br>efg</p>');
+                    window.chai.expect(clipboardData.getData('text/odoo-editor')).to.be.equal('<p>abc<br>efg</p>');
+                },
+            });
+            await testEditor(BasicEditor, {
+                contentBefore: '<p>[<span style="font-size: 16px;">Test</span> <span style="font-size: 48px;"><font style="color: rgb(255, 0, 0);">Test</font></span>]</p>',
+                stepFunction: async editor => {
+                    const clipboardData = new DataTransfer();
+                    triggerEvent(editor.editable, 'copy', { clipboardData });
+                    window.chai.expect(clipboardData.getData('text/plain')).to.be.equal('Test Test');
+                    window.chai.expect(clipboardData.getData('text/html')).to.be.equal('<p><span style="font-size: 16px;">Test</span> <span style="font-size: 48px;"><font style="color: rgb(255, 0, 0);">Test</font></span></p>');
+                    window.chai.expect(clipboardData.getData('text/odoo-editor')).to.be.equal('<p><span style="font-size: 16px;">Test</span> <span style="font-size: 48px;"><font style="color: rgb(255, 0, 0);">Test</font></span></p>');
                 },
             });
         });
```

Comment:
Can you please add a new "it" block with tests for copying list items? The goal is to have the current behavior written down as a spec.

- When we copy a single list item (or part of it), we want the `<li>` on the clipboard (and not the outer `<ul>`, `<ol>`, etc)
- When we copy a selection that spans multiple list items, we want a list (`<ul>`, `<ol>`, ...). 

I know this is not directly related to your task (these tests should have been written before...), but we have to make sure your fix does not change this behavior!

Reply:
Done!

Reply:
Nice! Thanks! 
I have just added 2 minor suggestions about reorganising the test cases, see comments below.